pkg/systemd/index.html

```diff
@@ -485,11 +483,8 @@ <h4 class="modal-title"></h4>
                     </table>
                 </div>
                 <div class="modal-footer">
-                    <button class="btn btn-default" translatable="yes" data-dismiss="modal">
-                        Cancel
-                    </button>
-                    <button class="btn btn-danger">
-                    </button>
+                    <button class="btn btn-default btn-danger" translatable="yes" data-dismiss="modal">Cancel</button>
```

Comment:
Thanks I'll remove this. But unless something is actually wrong, I would suggest doing further fixes or missed translations as additional follow ups.